hastexo/hastexo.py

```diff
@@ -163,31 +181,25 @@ def get_block_ids(self):
 
         return (course_id, student_id)
 
-    def get_stack_template(self):
+    def read_from_contentstore(self, path):
         """
-        Load the stack template directly from the course's content store.
-
-        Note: accessing the contentstore directly is not supported by the
-        XBlock API, so this depends on keeping pace with changes to
-        edx-platform itself.  Because of it, this should be replaced with an
-        HTTP GET to the LMS, in the future.
+        Loads a file directly from the course's content store.
 
         """
         course_id, _ = self.get_block_ids()
-        stack_template = None
+        contents = None
         try:
             from xmodule.contentstore.content import StaticContent
             from xmodule.contentstore.django import contentstore
             from xmodule.exceptions import NotFoundError
 
-            loc = StaticContent.compute_location(course_id,
-                                                 self.stack_template_path)
+            loc = StaticContent.compute_location(course_id, path)
             asset = contentstore().find(loc)
-            stack_template = asset.data
+            contents = asset.data
         except (ImportError, NotFoundError):
             pass
```

Comment:
Do we really want to succeed silently on failure here? Not even a log message if the course author fails to include a template, or botches the reference?